mediaflow_proxy/routes/extractor.py

```
@@ -1,61 +1,61 @@
+        if extractor_params.redirect_stream:
+            stream_url = encode_mediaflow_proxy_url(
+                **response,
+                response_headers=proxy_headers.response,
+            )
+            return RedirectResponse(url=stream_url, status_code=302)
+
+        return response
+
+    except ExtractorError as e:
+        logger.error(f"Extraction failed: {str(e)}")
+        raise HTTPException(status_code=400, detail=str(e))
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)